tests/integration/targets/fusion_ni/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+# TODO: Currently not possible to remove network interface group from network interface
+
+# Prepare
+- name: Create new network interface group foo in AZ bar
+  purestorage.fusion.fusion_nig:
+    name: "interface_group1"
+    availability_zone: az1
+    region: pure-us-west
+    state: present  # or absent
+    mtu: 1500
+    gateway: 172.17.1.1
+    prefix: 172.17.1.0/24
+  environment: "{{ test_env }}"
+
```

Comment:
We have [network_interfaces](https://github.com/Pure-Storage-Ansible/Fusion-Collection/blob/a8c5b152f65703cffccd8703b27385de4e5e42de/plugins/modules/fusion_info.py#L1082) in fusion_info module .  We can get them to check if they exist like we do in other tests

Reply:
Yes, I am aware. I cannot use it with double-agent control plane though, because it contains Fiber Channel interface and our SDK does not work with it.  It's not really a bug, FC is not supported at the moment.